test_scripts/PROJECTION/000_Start_secured_video_audio_service.lua

```diff
@@ -0,0 +1,63 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) Application is registered with PROJECTION appHMIType
+-- 2) and starts secured video and audio services
+-- SDL must:
+-- 1) Start services successful
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/PROJECTION/common')
+local runner = require('user_modules/script_runner')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Variables ]]
+local appHMIType = "PROJECTION"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appName = "server"
+config.application1.registerAppInterfaceParams.appID = "SPT"
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.defaultProtocolVersion = 3
```

Comment:
@GetmanetsIrina Don't agree with the previous comment. 3rd protocol is defaulted in ```common``` module for __these__ scripts. So we don't need to define it in any script explicitly.

Reply:
Agree. Locally changed.